pkg/detectors/gitlab/gitlab.go

```diff
@@ -53,7 +53,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 			resMatch = keyString[len(keyString)-1]
 		}
 
-		secret := detectors.Result{
+		s1 := detectors.Result{
```

Comment:
Did you rename this for similarity to other detectors?

Reply:
yes, any detectors that i updated